nncf/torch/dynamic_graph/wrappers.py

```diff
@@ -278,8 +278,16 @@ def wrap_parameters(model: torch.nn.Module):
 
     :param model: A model.
     """
+    from nncf.torch.external_hook import EXTERNAL_OP_STORAGE_PREFIX
+    from nncf.torch.quantization.external_quantizer import EXTERNAL_QUANTIZERS_STORAGE_PREFIX
+
+    # Ignore modules that will be added by NNCF
+    ignored_prefixes = [EXTERNAL_QUANTIZERS_STORAGE_PREFIX, EXTERNAL_OP_STORAGE_PREFIX]
+
     ctx = get_current_context()
     for name, param in model.named_parameters():
+        if any(name.startswith(ignore_prefix) for ignore_prefix in ignored_prefixes):
```

Comment:
Could you think of a way to get this information from the model?

Reply:
Looks like no.
It can be done by excluding all parameters in model.nncf, but in this case better to exclude by `name.startswith("_nncf")` condition. but in this case

Reply:
It make sense to exclude all parameters in model.nncf. Could you implement it?

Reply:
done